app/assets/styles/partials/components/_panel.scss

```diff
@@ -1,4 +1,18 @@
-.alert {
+@mixin panel_type($name, $color, $color-bg) {
+  .panel--#{$name} {
+    background: $color-bg;
```

Comment:
.panel__header has a background colour which produces this:
![screen shot 2016-12-13 at 11 41 44](https://cloud.githubusercontent.com/assets/130312/21139251/db564cfe-c129-11e6-9682-74923e027bee.png)